echo/server/dembrane/api/conversation.py

```
@@ -594,4 +639,48 @@ async def delete_conversation(
         raise HTTPException(
             status_code=500,
             detail=f"Failed to delete conversation: {str(e)}"
-        ) from e
\ No newline at end of file
+        ) from e
+
+@ConversationRouter.get("/health/stream")
+async def stream_health_data(
+    request: Request,
+    conversation_ids: Optional[List[str]] = Query(None),
```

Ruff (0.11.9)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)